webextension/popup.html

```diff
@@ -36,25 +36,45 @@ <h2>
       </h2>
       <p></p>
       <form>
-        <select name="issueType" required>
+        <select name="type" required>
           <option value="" hidden></option>
           <option value="desktopNotMobile"></option>
           <option value="siteUnusable"></option>
           <option value="brokenDesign"></option>
           <option value="playbackFailure"></option>
           <option value="other"></option>
         </select>
-        <button id="issueTakeScreenshot"></button>
-        <div id="issueRemoveScreenshot">
+        <button class="takeScreenshot"></button>
+        <div class="screenshot">
           <p></p>
           <img src="screenshot.jpg" alt="" />
           <button></button>
         </div>
-        <textarea name="issueDescription" rows="1"></textarea>
+        <textarea name="description" rows="1"></textarea>
       </form>
-      <label>
-        <input type="checkbox" name="includeURL" checked />
-      </label>
+      <a href="#" data-action="showFeedbackDetails"></a>
```

Comment:
We should either make this a `<button>` and style it like a link, or add `role="button"`.

Reply:
Do you have a preference? (I'll otherwise just add the `role`, as that's simplest)

Reply:
role seems easiest!